source/install/test_cc.sh

```diff
@@ -29,7 +29,7 @@ make -j${NPROC}
 
 #------------------
 cd ${SCRIPT_PATH}/../api_cc/tests
-${BUILD_TMP_DIR}/runUnitTests
+LD_LIBRARY_PATH=${BUILD_TMP_DIR}:$LD_LIBRARY_PATH ${BUILD_TMP_DIR}/runUnitTests
```

Comment:
@denghuilu Do you know how to set `RPATH` for `api_cc/tests/runUnitTests`, instead of setting runtime `LD_LIBRARY_PATH`?

Reply:
@caic99 Any idea?

Reply:
Resolved by installing `runUnitTests`.